SemanticUI/Collections/Message.elm

```diff
@@ -108,45 +127,46 @@ view extraAttributes cfg contents =
         )
         (case cfg.icon of
             Nothing ->
-                contentWithHeader
+                closeContent ++ allContent
 
             Just a ->
+                closeContent ++
```

Comment:
This is in both branches so I think we should move it outside the `case` statement.